assets/js/components/ErrorText.js

```diff
@@ -67,17 +68,20 @@ function ErrorText( { message, reconnectURL } ) {
 			<p
 				dangerouslySetInnerHTML={ sanitizeHTML( error, sanitizeArgs ) }
 			/>
+			{ children && <Fragment>&nbsp;{ children }</Fragment> }
 		</div>
 	);
 }
 
 ErrorText.propTypes = {
 	message: PropTypes.string.isRequired,
 	reconnectURL: PropTypes.string,
+	children: PropTypes.element,
 };
 
 ErrorText.defaultProps = {
```

Comment:
@sashadoes See my previous comment about deprecated defaultProps. Let's remove it :)